src/main/java/org/folio/list/services/UserFriendlyQueryService.java

```diff
@@ -100,6 +101,11 @@ private String handleNotContains(NotContainsCondition notContainsCondition, Enti
     return handleConditionWithPossibleIdValue(notContainsCondition, entityType, "does not contain", labelFn);
   }
 
+  private String handleEmpty(EmptyCondition emptyCondition) {
+    boolean isEmpty = Boolean.TRUE.equals(emptyCondition.value());
+    return isEmpty ? emptyCondition.fieldName() + " is empty" :  emptyCondition.fieldName() + " is not empty";
```

Comment:
split this across 2 lines just for readability

Reply:
maybe make it a regular if/else for readability, too?